main.py

```diff
@@ -220,11 +220,15 @@ def vms(self):
         vms = []
 
         for line in out.splitlines():
-            m = re.search(r'"([a-zA-Z0-9-_]+)"', line)
-            if m:
-                name = m.group(1)
+            m = re.findall(r'([a-zA-Z0-9-_]+)', line)
+            if m[0] != "inaccessible":
+                name = m[0]  # m.group(1)
                 state = self.state(name)
                 vms.append((state, name))
+            else:
+                name = m[1]
+                state = "inaccessible"  # self.state(name)
+                vms.append((state, name))
```

Comment:
If your condition ends with the same expression anyway (`vms.append((state, name))`), you may take it outside the condition so you won't repeat code.

Reply:
fixed